.evergreen/spec.patch

```diff
@@ -1,15 +1,19 @@
---- mongo-cxx-driver.spec.orig	2020-08-17 16:22:55.490400655 -0400
-+++ mongo-cxx-driver.spec	2020-08-17 16:26:24.490702714 -0400
-@@ -1,6 +1,8 @@
+--- mongo-cxx-driver.spec.orig	2021-07-21 13:05:46.139038889 -0400
++++ mongo-cxx-driver.spec	2021-07-22 18:23:45.099314489 -0400
+@@ -1,17 +1,17 @@
  # for better compatibility with SCL spec file
  %global pkg_name mongo-cxx-driver
  
 +%define _unpackaged_files_terminate_build 0
```

Comment:
Is this now required to account for the fact that tests are no longer built?

Reply:
I think you are misreading the change.  The `spec.patch` file is itself a diff, so we are looking at a diff of a diff.  The `+%define _unpackaged_files_terminate_build 0` line is just context.  It is not a change related to my commit.  That said, the reason for that directive is that our build creates both shared and static libraries, but the RPM only ships the shared libraries.  This leaves the static libraries in the install tree without them ending up in an RPM at the end of the process.  By default, the RPM build process will fail when this happens, so that the maintainer does not inadvertently leave parts of the package behind.  Remi could probably also just set `-DBUILD_SHARED_AND_STATIC_LIBS=OFF` so that the static libraries are not built.  But either approach is fine.

Reply:
🤦 you are totally right – I thought this was introduced here. Thank you for the additional background. I hadn't realized that RPM only ships shared libraries.